### PR TITLE
Readme: change intro question

### PR DESCRIPTION
**Pull request by jotak**
I think it's more correct to reformulate the question, because Kiali doesn't know anything about what a service is doing, beyond its interactions. We have no idea of the services business value.

See also https://github.com/kiali/kiali.io/pull/20

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -6,7 +6,7 @@ toc::[]
 
 == Introduction
 
-Kiali provides answers to the question _What are the microservices in my Istio service mesh doing?_
+Kiali provides answers to the questions: _What are the microservices in my Istio service mesh, and how are they connected?_
```
Comment (abonas) [inline README.adoc]: Maybe "What microservices exist in my Istio service mesh" instead of "what are ....", otherwise I feel it misses a verb :)

Comment (JStickler) [reply README.adoc]: I would choose either "What microservices are part of my service mesh and how are they connected."  or "Which microservices exist in my service mesh and how are they connected."

**Comment by pilhuhn** [comment]
I'd merge that now, as the counterpart on kiali.io has been merged.

File: README.adoc (hunk 1 above)
Comment (abonas) [reply README.adoc]: @jotak have you addressed @JStickler comments?

Comment (jotak) [reply README.adoc]: done

**Pull request merged**